### ISSUE TITLE
TypeInitializationException again

### ISSUE THREAD

**Issue by matteopey** [PowerShell/PSReadLine#787]
I'm getting exactly the same error as #614 but with a local build from the latest master of PSReadLine that I built using the ENG keyboard layout, and then use the ITA layout to test.

Environment data
----------------

```
PS version: 5.1.17763.1
PSReadline version: 2.0.0-beta3
os: 10.0.17763.102 (WinBuild.160101.0800)
PS file version: 10.0.17763.102 (WinBuild.160101.0800)
```

Steps to reproduce or exception report
--------------------------------------
```
❯ Get-PSReadLineOption
Get-PSReadLineOption : L'inizializzatore di tipo di 'Microsoft.PowerShell.PSConsoleReadLine' ha generato un'eccezione.
In riga:1 car:1
+ Get-PSReadLineOption
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-PSReadLineOption], TypeInitializationException
    + FullyQualifiedErrorId : System.TypeInitializationException,Microsoft.PowerShell.GetPSReadLineOption
```

**Comment by lzybkr** [comment]
Sorry about that, thanks for providing enough information for me to repro. Fix is in #788.

**Comment by lzybkr** [comment]
@matteopey - no problem, this project has always been mostly a nights/weekends project.
